<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TDialogLocate</class>
 <widget class="QDialog" name="TDialogLocate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_text">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>设置文字</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_column">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>应用后列增</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_row">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>应用后行增</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_col">
    <property name="text">
     <string>定位列号</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_column"/>
   <widget class="QPushButton" name="pushButton_apply">
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_row">
    <property name="text">
     <string>定位行号</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_row"/>
   <widget class="QPushButton" name="pushButton_quit">
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_quit</sender>
   <signal>clicked()</signal>
   <receiver>TDialogLocate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
